rebar.config

```diff
@@ -23,7 +23,7 @@
          {git, "git://github.com/chef/pooler.git", {branch, "master"}}},
 
         {envy, ".*",
-         {git, "git://github.com/manderson26/envy.git", {branch, "master"}}}
+         {git, "git://github.com/markan/envy.git", {branch, "master"}}}
```

Comment:
This should be chef/envy right?

Reply:
There's no chef envy :)  So, I'm making an issue to fork Mark's repo.